oneview/resource_logical_interconnect_group.go

```diff
@@ -1932,17 +2296,30 @@ func resourceLogicalInterconnectGroupRead(d *schema.ResourceData, meta interface
 	for i, snmpAccessIP := range logicalInterconnectGroup.SnmpConfiguration.SnmpAccess {
 		snmpAccess[i] = snmpAccessIP
 	}
+	// snmpUsers := make([]interface{}, len(logicalInterconnectGroup.SnmpConfiguration.SnmpUsers))
+	// for _, snmpUsersIP := range logicalInterconnectGroup.SnmpConfiguration.SnmpUsers {
```

Comment:
If it is not required, removed these commented code

Reply:
Done